js/pull-requests.js

```diff
@@ -0,0 +1,95 @@
+// fetch("https://api.github.com/repos/rarmatei/js-exercises/pulls")
+//   .then(function(response) {
+//     return response.clone().json();
+//   })
+//   .then(function(el) {
+//     function getPullRequestByUser() {
+//       document.querySelector("#pull-requests-list").innerHTML = "";
+//       var user = document.querySelector("#searchField").value;
+//       el.filter(element => element.user.login === user).forEach(element => {
+//         var pullRequest = document.createElement("li");
+//         document.querySelector("#pull-requests-list").appendChild(pullRequest);
+//         var a = document.createElement("a");
+//         a.setAttribute("href", element.html_url);
+//         a.innerHTML = element.title;
+//         pullRequest.appendChild(a);
+//       });
+//     }
+
+//     document
+//       .querySelector("#searchField")
+//       .addEventListener("keyup", getPullRequestByUser);
+//   })
+//   .catch(function(error) {
+//     console.log(error);
+//   });
+
+// fetch("https://api.github.com/repos/rarmatei/js-exercises/pulls")
+//   .then(function(response) {
+//     return response.clone().json();
+//   })
+//   .then(function(el) {
+//     function createLinkForAllPullRequests(jsonData) {
+//       jsonData.forEach(element => {
+//         var pullRequest = document.createElement("li");
+//         document.querySelector("#pull-requests-list").appendChild(pullRequest);
+//         var a = document.createElement("a");
+//         a.setAttribute("href", element.html_url);
+//         a.innerHTML = element.title + " by " + element.user.login;
+//         pullRequest.appendChild(a);
+//       });
+//     }
+
+//     document.querySelector("#pull-requests-list").innerHTML = "";
+//     createLinkForAllPullRequests(el);
+//           var user = document.querySelector("#searchField").value;
+//     document
+//       .querySelector("#searchField")
+//       .addEventListener("keyup", function(e) {
+//         var valueOfInputField = e.target.value.toLowerCase();
+//        el.filter(function())
+//       });
+//   });
+
+fetch("https://api.github.com/repos/rarmatei/js-exercises/pulls")
+  .then(function(response) {
+    return response.clone().json();
+  })
+  .then(function(el) {
+    document.querySelector("#pull-requests-list").innerHTML = "";
+    //var user = document.querySelector("#searchField").value.toLowerCase();
+    el.forEach(creatLink);
+    function creatLink(element) {
```

Comment:
`el` and `element` aren't good names to use here, particularly since we are using HTML and DOM here which means someone reading the code will think you are talking about [HTML elements](https://www.w3schools.com/html/html_elements.asp). So I would expect something returned by a DOM function function like `document.createElement` to be an "element" in this context, but not the result of your fetch query (which is a list of objects at this point).